tests/cpu_esp_i2c_hw/README.md

```diff
@@ -0,0 +1,17 @@
+# ESP Hardware I2C Test
+
+## About
+
+The application corresponds exactly to the application `tests/periph_i2c`.
+It is only for testing the timer implementation using hardware counters.
+In fact, the application is nothing more than the `tests/periph_i2c`
+application compiled with the following command:
```

Comment:
What timer implementation gets used on the esp for `tests/periph_i2c`?

Reply:
The hardware implementation `esp_i2c_hw` doesn't require a timer. The bit-banging software implementation uses busy waiting.

Reply:
Why would you default to bit-banging when there is a hardware i2c?

Reply:
Because it is buggy, instable and slower than the software implementation.